Maker-Agents-Suit-Module/OSMAgent/src/priceUpdateCheck.ts

```diff
@@ -0,0 +1,76 @@
+import BigNumber from "bignumber.js";
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import TimeTracking from "./TimeTracking";
+import {
+  provideFunctionCallsDetectorAgent,
+  FindingGenerator,
+} from "@nethermindeth/general-agents-module";
+
+const time = new TimeTracking();
+const address = "0x2417c2762ec12f2696f62cfa5492953b9467dc81";
+export const functionSignature = "function poke()";
+
+const createFindingGenerator = (alertId: string): FindingGenerator => {
+  return (metadata: { [key: string]: any } | undefined): Finding => {
+    return Finding.fromObject({
+      name: "Method not called within the first 10 minutes",
+      description: "Poke() function not called within 10 minutes of the hour",
+      alertId: "NETHFORTA-24",
```

Comment:
We are using other convention for these agents.
```suggestion
      alertId: "MakerDAO-OSM-4",
```